### PR TITLE
Task #79034 - Upgrade packages

### PR DESCRIPTION
**Pull request by tabman83**

### CODE REVIEW

File: src/content/src/WebAPIService.Fabric/WebAPIService.Fabric.sfproj [hunk 1]
```diff
@@ -1,11 +1,11 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <Import Project="..\packages\Microsoft.VisualStudio.Azure.Fabric.MSBuild.1.6.9\build\Microsoft.VisualStudio.Azure.Fabric.Application.props" Condition="Exists('..\packages\Microsoft.VisualStudio.Azure.Fabric.MSBuild.1.6.9\build\Microsoft.VisualStudio.Azure.Fabric.Application.props')" />
+  <Import Project="..\packages\Microsoft.VisualStudio.Azure.Fabric.MSBuild.1.7.1\build\Microsoft.VisualStudio.Azure.Fabric.Application.props" Condition="Exists('..\packages\Microsoft.VisualStudio.Azure.Fabric.MSBuild.1.7.1\build\Microsoft.VisualStudio.Azure.Fabric.Application.props')" />
```
Comment (chomick) [inline src/content/src/WebAPIService.Fabric/WebAPIService.Fabric.sfproj:3]: Are you happy those changes will not impact existing pipelines (verified with DevOps)?

File: src/content/src/WebAPIService.Fabric/WebAPIService.Fabric.sfproj [hunk 2]
```diff
@@ -1,11 +1,11 @@
...
   <PropertyGroup Label="Globals">
     <ProjectGuid>bee428ac-ffdd-4af6-8a01-b72d8eed875b</ProjectGuid>
     <ProjectVersion>2.0</ProjectVersion>
     <MinToolsVersion>1.5</MinToolsVersion>
-    <SupportedMSBuildNuGetPackageVersion>1.6.4</SupportedMSBuildNuGetPackageVersion>
+    <SupportedMSBuildNuGetPackageVersion>1.7.1</SupportedMSBuildNuGetPackageVersion>
```
Comment (chomick) [inline src/content/src/WebAPIService.Fabric/WebAPIService.Fabric.sfproj:8]: Are you happy those changes will not impact existing pipelines (verified with DevOps)?

File: src/content/src/WebAPIService.Fabric/WebAPIService.Fabric.sfproj (hunk 1 above)
Comment (tabman83) [reply src/content/src/WebAPIService.Fabric/WebAPIService.Fabric.sfproj:3]: ah - good point - what's the best guy to ask to? (Colin's off)

File: src/content/src/WebAPIService/WebAPIService.csproj [hunk 3]
```diff
@@ -11,23 +11,23 @@
...
+    <PackageReference Include="Autofac" Version="5.2.0" />
     <PackageReference Include="Autofac.Extensions.DependencyInjection" Version="6.0.0" />
-    <PackageReference Include="Eshopworld.DevOps" Version="5.0.1" />
-    <PackageReference Include="Eshopworld.Telemetry" Version="3.1.2" />
-    <PackageReference Include="Eshopworld.Web" Version="4.0.1" />
-    <PackageReference Include="IdentityServer4" Version="3.1.2" />
+    <PackageReference Include="Eshopworld.DevOps" Version="5.1.4" />
+    <PackageReference Include="Eshopworld.Telemetry" Version="3.1.5" />
+    <PackageReference Include="Eshopworld.Web" Version="4.1.2" />
+    <PackageReference Include="IdentityServer4" Version="4.0.4" />
```
Comment (chomick) [inline src/content/src/WebAPIService/WebAPIService.csproj:24]: Can you verify with Kevin MacUistin this is supported/correct, please?

Comment (macuistin) [reply src/content/src/WebAPIService/WebAPIService.csproj:24]: We're still on IdentityServer 3.1.3. Looks like I forgot to check in the last 2 months.
Given that IdentityServer4 is standards-based, I would assume that this should not be an issue.
Please confirm.

Comment (tabman83) [reply src/content/src/WebAPIService/WebAPIService.csproj:24]: This is a template so it's not easy to have a comprehensive set of tests on this. To be honest I would be more comfortable in downgrading it unless you are absolutely sure.

Comment (macuistin) [reply src/content/src/WebAPIService/WebAPIService.csproj:24]: So when you update the template you don't create a test API to verify it?

Comment (tabman83) [reply src/content/src/WebAPIService/WebAPIService.csproj:24]: There's an API name 'WebAPIService' in settings with no scopes which I believe it's not configured either so I have no immediate way to test the authentication. I can put in details from a different API just for the sake of testing the authorization, but then will that be enough to call it fine on the package upgrade?

Comment (macuistin) [reply src/content/src/WebAPIService/WebAPIService.csproj:24]: Yes, as I will plan to upgrade package versions in the near future

**Comment by tabman83** [comment]
Closing for now as the upgrade of SF packages won't work on some company build machines.

**Pull request closed without merging**